### PR TITLE
Improve data docs for node sdk 2.0.2

### PR DESCRIPTION
**Pull request by ThyeeZz**

### CODE REVIEW

**Review by YiyunNi — APPROVED** [review]

File: API_Reference/milvus-sdk-node/v2.0.2/Data/deleteEntities.md [hunk 1]
```diff
@@ -1,28 +1,39 @@
...
  });
  ```
  
 ## Return
 ```javascript
 // DeleteEntitiesReq return
+{
+  succ_index: [],
+  err_index: [],
+  status: { error_code: 'Success', reason: '' },
```
Comment (nameczz) [inline API_Reference/milvus-sdk-node/v2.0.2/Data/deleteEntities.md:31]: status  first line

File: API_Reference/milvus-sdk-node/v2.0.2/Data/insert.md [hunk 2]
```diff
@@ -1,34 +1,60 @@
...
 ### note 
-If the field type is binary, the vector data length needs to be dimension / 8
+If the field type is binary, the vector data length needs to be dimension / 8.
 
 ## Example
 ```javascript
+
+const vectorsData = Array.from({ length: 10 }).map(() => ({
+    vector_01: Array.from({ length: 4 }).map(() =>
+      Math.floor(Math.random() * 100000)
```
Comment (nameczz) [inline API_Reference/milvus-sdk-node/v2.0.2/Data/insert.md:26]: 100000 is too large .. change to 10

File: API_Reference/milvus-sdk-node/v2.0.2/Data/insert.md [hunk 3]
```diff
@@ -1,34 +1,60 @@
...
 ## Return
 ```javascript
 // insert return
+{
+  succ_index: [
+     0,  1,  2,  3,  4,  5,  6,  7,  8,  9,
+    ... 990 more items
+  ],
+  err_index: [],
+  status: { error_code: 'Success', reason: '' },
```
Comment (nameczz) [inline API_Reference/milvus-sdk-node/v2.0.2/Data/insert.md:44]: status first line

File: API_Reference/milvus-sdk-node/v2.0.2/Client/closeConnection.md [hunk 4]
```diff
@@ -17,4 +17,6 @@ milvusClient.closeConnection();
 # Return
 ```javascript
 // closeConnection return
```
Comment (nameczz) [inline API_Reference/milvus-sdk-node/v2.0.2/Client/closeConnection.md:19]: remote / closeConnection return

**Pull request merged**